x-pack/plugins/maps/public/connected_components/mb_map/transform_request.ts

```diff
@@ -43,7 +61,7 @@ export function transformRequest(url: string, resourceType: string | undefined)
     };
   }
 
-  if (resourceType === 'Tile' && url.startsWith(GETGRIDTILE)) {
+  if (resourceType === 'Tile' && pathOrUrl.startsWith(GETGRIDTILE)) {
```

Comment:
We do not have to test the entire URL. If its easier, we could just test that URL contains path `MVT_GETGRIDTILE_API_PATH`